docs/kubeadm.md

```diff
@@ -0,0 +1,249 @@
+    Name:                   front-end
+    Namespace:              default
+    Labels:                 name=front-end
+    Selector:               name=front-end
+    Type:                   NodePort
+    IP:                     100.66.88.176
+    Port:                   <unset> 80/TCP
+    NodePort:               <unset> 31869/TCP
+    Endpoints:              <none>
+    Session Affinity:       None
+
+It will take several minutes to download and start all the containers, watch the output of `kubectl get pods` to see when they're all up and running.
+
+Then go to the IP address of your cluster's master node in your browser, and specify the given port.
+So for example, `http://<master_ip>:<port>`.
+In the example above, this was `31869`, but it will be a different port for you.
+
+If there is a firewall, make sure it exposes this port to the internet before you try to access it.
+
+### Explore other add-ons
+
+See the [list of add-ons](/docs/admin/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization &amp; control of your Kubernetes cluster.
+
+
+## What's next
+
+* Learn more about [Kubernetes concepts and kubectl in Kubernetes 101](/docs/user-guide/walkthrough/).
+* Install Kubernetes with [a cloud provider configurations](/docs/getting-started-guides/) to add Load Balancer and Persistent Volume support.
+
+
+## Cleanup
+
+* To uninstall the socks shop, run `kubectl delete -f https://raw.githubusercontent.com/lukemarsden/microservices-demo/master/deploy/kubernetes/definitions/wholeWeaveDemo-NodePort.yaml`.
+* To uninstall Kubernetes, simply delete the machines you created for this tutorial.
+  Or alternatively, uninstall the `kubelet`, `kubeadm` and `kubectl` packages and then manually delete all the Docker container that were created by this process.
+
+## How to provide feedback
+
+* Slack Channel: [#sig-cluster-lifecycle](https://kubernetes.slack.com/messages/sig-cluster-lifecycle/)
+* Mailing List: [kubernetes-sig-cluster-lifecycle](https://groups.google.com/forum/#!forum/kubernetes-sig-cluster-lifecycle)
+* [GitHub Issues](https://github.com/kubernetes/kubernetes/issues): please tag `kubeadm` issues with `@kubernetes/sig-cluster-lifecycle`
+
+## Limitations
+
+1. The cluster we create here won't have cloud-provider integrations, so for example won't work with (for example) [Load Balancers](/docs/user-guide/load-balancer/) (LBs) or [Persistent Volumes](/docs/user-guide/persistent-volumes/walkthrough/) (PVs).
+
+   Instead we will use the [NodePort feature of services](/docs/user-guide/services/) to demonstrate exposing the sample application on the internet.
+   To easily obtain a cluster which works with LBs and PVs Kubernetes, try [the "hello world" GKE tutorial](/docs/hellonode) or [one of the other cloud-specific installation tutorials](/docs/getting-started-guides/).
+1. The cluster we create here will have a single master, with a single `etcd` database running on it.
```

Comment:
Maybe explain or link to the limitations. When does data loss occur, when is HA needed.